<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionPasteWindow</class>
 <widget class="QMainWindow" name="SelectionPasteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="layout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTextBrowser" name="previewBrowser"/>
      <widget class="DetailView" name="listView"/>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitButton">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DetailView</class>
   <extends>QListView</extends>
   <header>ui/detailview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
